cmd/sensorbee/default_main.go

```diff
@@ -20,8 +21,8 @@ func init() {
 func main() {
 	app := cli.NewApp()
 	app.Name = "sensorbee"
-	app.Usage = "SensorBee"
-	app.Version = "0.3.2" // TODO: don't hardcode the version number
+	app.Usage = "SensorBee built with build_sensorbee 0.4.1"
```

Comment:
No, it shouldn't be. If we do it, we cannot see the problem when we build a sensorbee binary with a wrong version of build_sensorbee because it always print the latest version at the time of the execution of build_sensorbee. For example, when a user wrongly uses build_sensorbee 0.1.0 and the version of installed sensorbee package is 0.2.0, the original approach prints 0.1.0 but your approach shows the binary is built with build_sensorbee 0.2.0 although the actual version of build_sensorbee is 0.1.0.